.github/workflows/backport.yml

```diff
@@ -26,15 +26,17 @@ jobs:
           # history is needed to run git cherry-pick below
           fetch-depth: 0
 
-      - name: Use CLA approved github bot
-        run: .github/scripts/use-cla-approved-github-bot.sh
```

Comment:
does this script stick around then?

Reply:
yeah, I'm only converting this one workflow for now, will convert them all (and probably use this script) once things are looking good